bundles/org.jkiss.utils/tests/org/jkiss/utils/ArrayUtilsTest.java

```diff
@@ -0,0 +1,170 @@
+package org.jkiss.utils;
+
+import org.jkiss.utils.ArrayUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayUtilsTest {
+
+  @Test
+  public void testIsEmpty() {
+    Object[] arr = null;
```

Comment:
Why not inline?

Reply:
It won't compile because it doesn't know which version of the `isEmpty` function to use.